docs/roadmap.md

```diff
@@ -32,14 +32,14 @@ As an unpackaged desktop app (WPF, WinForms, Win32, or console, not using [MSIX]
  🌟 |       Reunion 0.5 (Q1 2021)      |         Reunion 0.8 (Q2 2021)   |       Reunion 1.0 (Q4 2021)    | 
 -- | ---------------------------------- | ------------------------------ | ------------------------------- |
```

Comment:
Consider using the Markdown table-cell-alignment markers so everything is left/right aligned without you manually adding spaces.  See https://docs.github.com/en/github/writing-on-github/organizing-information-with-tables#formatting-content-within-your-table

Reply:
@jonwis the manual spaces here are purely for ease of viewing when as raw markdown, they have no effect on the visual alignment of the final rendered table (markdown trims all those spaces).

Including spaces in the markdown is a stylistic choice, as I wasn't the author of these tables, I won't change anything there.